subprojects/gradle/runner/client/src/main/kotlin/com/avito/runner/scheduler/metrics/TestKey.kt

```diff
@@ -0,0 +1,8 @@
+package com.avito.runner.scheduler.metrics
+
+import com.avito.runner.service.model.TestCase
+
+internal data class TestKey(
```

Comment:
The "key" looks like a usage case.
Maybe it's better to express the content itself: `TestCaseExecution` or similar.